styles/sass/variables/_colors.scss

```diff
@@ -78,6 +78,10 @@ $tableBackgroundAccent:         var(--sd-item__main-Bg--alternate); // for strip
 $tableBackgroundHover:          var(--sd-item__main-Bg--hover); // for hover
 $tableBorder:                   var(--sd-colour-line--medium); // table and cell border
 
+// Icon circle
+// -------------------------
+$iconBackground: hsl(168, 33%, 55%);
```

Comment:
use a CSS variable instead of a SASS variable. Also use one of the colors that are alreade defined.